### DISCUSSION TITLE
pc blue screening

### DISCUSSION THREAD

**Discussion by Sebleptic** [Atlas-OS/Atlas#1304] [Help & Support]
So i installed Atlas os on my test pc laptop with an 3060 and i7 12 gen and it just keeps blue screening and saying "MEMORY MANAGEMENT" anyone know what this is and how to fix?

**Comment by omardiaadev** [comment]
It's highly unlikely that `MEMORY MANAGEMENT`'s BSOD is caused by AtlasOS.
This however could be caused by either faulty memory (Hardware Problem) or you've installed drivers could potentially cause this error.

Doing a quick search on the internet, I've found this [Microsoft article](https://answers.microsoft.com/en-us/windows/forum/all/solved-memory-management-error-not-fixing-please/dad5eb25-0229-4c8c-9d28-064c97ec8355) that could help you:
https://answers.microsoft.com/en-us/windows/forum/all/solved-memory-management-error-not-fixing-please/dad5eb25-0229-4c8c-9d28-064c97ec8355
